cg/server/api.py

```diff
@@ -68,7 +67,18 @@
 @BLUEPRINT.route("/submit_order/<order_type>", methods=["POST"])
 def submit_order(order_type):
     """Submit an order for samples."""
-    api = OrdersAPI(lims=lims, status=db, osticket=osticket, freshdesk_client=freshdesk_client)
+    ticket_handler = TicketHandler(
+        status_db=db,
+        base_url=ticket_service.base_url,
+        api_key=ticket_service.api_key,
+        order_email_id=ticket_service.order_email_id,
+        env=ticket_service.env,
+    )
+    api = OrdersAPI(
```

Comment:
The `OrdersAPI` should not be instantiated in the endpoint. Create it in the `ext.py` module and import it